WalletWasabi.Fluent/Views/Dialogs/ManualCoinJoinProfileDialogView.axaml

```diff
@@ -13,26 +13,51 @@
     CancelContent="Cancel"
     EnableCancel="{Binding EnableCancel}"
     EnableBack="{Binding EnableBack}"
-    EnableNext="True" NextContent="Done">
+    EnableNext="True" NextContent="Done"
+    Padding="20 0 20 0">
     <StackPanel Classes="settingsLayout">
-      <DockPanel>
+
+      <DockPanel IsVisible="{Binding ShowAutomaticCoinjoin}">
         <TextBlock Text="Automatic Coinjoin" />
         <ToggleSwitch IsChecked="{Binding AutoCoinjoin}" />
       </DockPanel>
 
-      <StackPanel Spacing="10">
+      <StackPanel Spacing="10" IsVisible="{Binding AutoCoinjoin}">
+        <TextBlock Text="Automatic Coinjoin Threshold" />
+        <DockPanel>
+          <PathIcon DockPanel.Dock="Left" Foreground="{StaticResource PrivacyLevelMinimalBrush}"
+                    Data="{StaticResource privacy_minimal}"
+                    Margin="0 20 0 0" VerticalAlignment="Top"
+                    ToolTip.Tip="No automatic coinjoin if the non-private wallet balance is less than this." />
+          <DataValidationErrors DockPanel.Dock="Bottom" Owner="{Binding #PlebStopThresholdInput}" MinHeight="21" />
+          <c:CurrencyEntryBox Text="{Binding PlebStopThreshold, Mode=TwoWay}" Name="PlebStopThresholdInput"
+                              CurrencyCode="BTC"
+                              Watermark="0 BTC"
+                              IsApproximate="False"
+                              DockPanel.Dock="Bottom"
+                              Margin="10 -10 0 0" Classes="standalone" />
+          <Slider Minimum="0" Maximum="4" Value="{Binding PlebStopThresholdFactor}" Margin="10 0 0 0" VerticalAlignment="Center"
+                  TickFrequency="1" IsSnapToTickEnabled="True" />
```

Comment:
@soosr 

> Slider is just nonsense here.

Might have been a misunderstanding on my part, but we discussed it with @danwalmsley and we agreed on this design.
Do you suggest to remove the slider and keep the currency input alone?

> Should the pleb stop really be under the customize?

Same as above. The idea is that only the simple one-click UI was visible first-hand, and that all other controls would be available under the "advanced" options UI. I can of course move it back to the first screen if needed, but I don't think it would really fit there with the current design

Reply:
When I tried out the slider it felt totally useless and my first impression was that it wouldn't be helping anyone.

> The idea is that only the simple one-click UI was visible first-hand, and that all other controls would be available under the "advanced" options UI.

ACK for the UX. It was just weird because it's under the `customize` section, but the profiles don't really change the pleb threshold.